### PR TITLE
add introduction section for ISC'22 tutorial

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by klust** [comment]
Two links in the "Reproducilbility" subsection may need attention after assembling the tutorial. I assume the links with name "toolchain" and "easyconfig files" are meant to refer to sections in the terminology page?

**Comment by klust** [comment]
Two links in the "Reproducilbility" subsection may need attention after assembling the tutorial. I assume the links with name "toolchain" and "easyconfig files" are meant to refer to sections in the terminology page?

File: docs/2022-isc22/introduction.md [hunk 1]
```diff
@@ -1 +1,146 @@
...
+For that same reason, EasyBuild **optimizes software for the processor architecture of the build host**
+by default, via compiler options like ``-march=native`` (GCC), ``-xHost`` (Intel compilers), etc.
+This behaviour [can be changed via the ``--optarch`` configuration setting](https://docs.easybuild.io/en/latest/Controlling_compiler_optimization_flags.html).
+
+
+#### Reproducibility
+
+In addition to performance, **reproducibility of installations** is a core aspect of EasyBuild.
+
+Most software installations performed with EasyBuild use a **particular compiler <a href="#toolchains">toolchain</a>**,
```
Comment (ocaisa) [inline docs/2022-isc22/introduction.md:95]:
```suggestion
Most software installations performed with EasyBuild use a **particular compiler [toolchain](terminology.md#toolchains)**,
```

File: docs/2022-isc22/introduction.md [hunk 2]
```diff
@@ -1 +1,146 @@
...
+with which we aim to be in control over the build environment and avoid relying on tools and libraries
+provided by the operating system. For similar reasons, we try to **provide all required dependencies through EasyBuild** as well,
+with a few notable exceptions, like ``OpenSSL`` for security reasons, and Infiniband and GPU drivers which
+are too closely intertwined with the operating system.
+
+For both toolchains and dependencies, **fixed software versions** are specified in the
+<a href="#easyconfig-files">easyconfig files</a>. That way, easyconfig files can easily be shared with others:
```
Comment (ocaisa) [inline docs/2022-isc22/introduction.md:102]:
```suggestion
[easyconfig files](terminology.md#easyconfig-files). That way, easyconfig files can easily be shared with others:
```

**Review by ocaisa — CHANGES_REQUESTED** [review]
I think these should work

File: docs/2022-isc22/introduction.md (hunk 1 above)
Comment (boegel) [reply docs/2022-isc22/introduction.md]: > I think these should work

@ocaisa Verified and fixed in 50d7f43

**Review by ocaisa — APPROVED** [review]

**Pull request merged**